napari/components/_tests/test_scale_bar.py

```diff
@@ -5,3 +5,9 @@ def test_scale_bar():
     """Test creating scale bar object"""
     scale_bar = ScaleBarOverlay()
     assert scale_bar is not None
+
+
+def test_scale_bar_fixed_width():
+    """Test creating scale bar object with fixed width"""
+    scale_bar = ScaleBarOverlay(fixed_width=50)
+    assert scale_bar.length == 50
```

Comment:
my bad, only suggested half of it xD

```suggestion
def test_scale_bar_fixed_length():
    """Test creating scale bar object with fixed length"""
    scale_bar = ScaleBarOverlay(length=50)
    assert scale_bar.length == 50
```